Example/Auth/Sample/MainViewController.m

```diff
@@ -1754,6 +1762,24 @@ - (void)signInWithCustomToken {
   [self presentViewController:dataEntryViewController animated:YES completion:nil];
 }
 
+/** @fn signInWithCustomTokenAuthResult
+    @brief Signs the user in using a manually-entered custom token.
+ */
+- (void)signInWithCustomTokenAuthResult {
+  CustomTokenDataEntryViewControllerCompletion action =
+      ^(BOOL cancelled, NSString *_Nullable userEnteredTokenText) {
+        if (cancelled) {
+          [self log:@"CANCELLED:sign-in with custom token cancelled."];
+          return;
+        }
+
+        [self doSignInAndRetrieveDataWithCustomToken:userEnteredTokenText];
```

Comment:
Should we use weak ref to self?
The same as 'StaticContentTableViewCell'

Reply:
This is fine this way, weakself wouldn't impact it.